kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -74,10 +74,11 @@ def setUp(self):
                 'labels': {
                     'foo': 'bar', 'kubernetes_pod_operator': 'True',
                     'airflow_version': airflow_version.replace('+', '-'),
-                    'execution_date': '2016-01-01T0100000100-a2f50a31f',
-                    'dag_id': 'dag',
-                    'task_id': 'task',
-                    'try_number': '1'},
+                    # 'execution_date': '2016-01-01T0100000100-a2f50a31f',
+                    # 'dag_id': 'dag',
+                    # 'task_id': 'task',
+                    # 'try_number': '1'
```

Comment:
What's going on here?

Reply:
This was code that was merged in erroneously as it's actually the "monitor pods by label" PR I intend to merge later this week. I will delete the commented lines once I see how the next tests go.